recipe/meta.yaml

```diff
@@ -1,51 +1,62 @@
 {% set name = "category_encoders" %}
-{% set version = "2.1.0" %}
+{% set version = "2.2.2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 9d76a2d810e02ad9527bed9c70b01fcb0f2238778b5422128275b5ba7561ca64
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 79651dc3e8cd6749bc5d404c49a79e93469016d106736326842305e076136cb0
+    folder: dist
+  - url: https://github.com/scikit-learn-contrib/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: b8fa5f2c2f3294ed7847e9c5486566604a6a321308dfb5377440a7ca2a9a5b1f
+    folder: src
 
 build:
   noarch: python
   number: 0
   script:
-    - rm -rf examples  # [unix]
-    - rd /s /q examples  # [win]
-    - "{{ PYTHON }} -m pip install --no-deps --ignore-installed -vvv ."
+    - cd dist && {{ PYTHON }} -m pip install -vvv .
 
 requirements:
   host:
-    - python
+    - numpy >=1.14.0
```

Comment:
these are a side-effect of requiring import of the module itself during `setup.py`